docs/src/main/paradox/discovery.md

```diff
@@ -184,6 +184,7 @@ then binds the default Service Account to the `Role` by creating a `RoleBinding`
 Adjust as necessary.
 
 ```yaml
```

Comment:
Added because IntelliJ's Markdown editor gets very confused without it. Seemed like it might be worth trading a rogue blank line in the final output for a smoother editing experience for contributors, but I'll happily revert it if not.

Reply:
I think it's ok; though I also think all those examples should be eventually made into paradox (and exposed in documentation page), let's do that separately though